### ISSUE TITLE
[NuGet.org Bug]: [A11y] [Accessibility Insights FastPass] 

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#8737]
### Impact

It's more difficult to complete my work

### Describe the bug

Resources for this rule
More information about select-name
WCAG 1.3.1, WCAG 4.1.2

Path | select
-- | --

Snippet:
```
<select name="installation-instructions" class="installation-instructions-dropdown">
```

Fix ONE of the following:
- Form element does not have an implicit (wrapped) <label>
- Form element does not have an explicit <label>
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element has no title attribute
- Element's default semantics were not overridden with role="none" or role="presentation"

![image](https://user-images.githubusercontent.com/94054/128429879-38cc40a0-6c73-409e-8ce9-5315b9ca65f8.png)

### Repro Steps

Run Accessibility Insights Fast Pass on this page:
https://int.nugettest.org/packages/MySqlConnector/1.4.0-beta.1?preview=1

### Expected Behavior

Fast Pass passes.
